CHANGELOG.md

```diff
@@ -24,6 +24,12 @@ This release has an [MSRV] of 1.82.
 ### Changed
 
 - Breaking change: `Image` has been renamed to `ImageBrush`, which now consists of an `ImageData` and an `ImageSampler`. ([#117][], [#123][] by [@nicoburns][], [@DJMcNab][])
+- Breaking change:  `Blob`, `Font`, and `WeakBlob` have been moved to the new `raw_resource_handle` crate. As part of this move, `Font` has been renamed to `FontData` to match `ImageData`. ([#126][] by [@nicoburns][])
```

Comment:
I think we should split out the breaking change rename and the movement to Linebender Resource Handle (in addition, instead of "have been moved", we could use "are not re-exports from")

This also needs to update the name of the crate.

Reply:
Not in this case, as the commits in #129 are completely unique (primarily because they don't include the rename)